app/views/provider_interface/offer/ske_lengths/_lengths.html.erb

```diff
@@ -0,0 +1,5 @@
+<%= f.govuk_radio_buttons_fieldset key, legend: { text: "How long must their SKE course in #{subject} be?" } do %>
```

Comment:
I think you mentioned already that you're planning on moving these into translation files?